changes/5146.feature

```diff
@@ -0,0 +1,9 @@
+API Tokens: an alternative to API keys. Tokens can be created and
+removed on demand(check :ref:`api authentication>`) and there is no
+restriction on the maximum number of tokens per user. Consider using
+it instead of them API key and prefer the creation of a separate token for
+each use-case instead of sharing the same token between multiple
+clients. In original implementation API Tokens are JWT, but the goal
```

Comment:
```suggestion
restriction on the maximum number of tokens per user. Consider using
tokens instead of API keys and create of a separate token for
each use-case instead of sharing the same token between multiple
clients. By default API Tokens are JWT, but the goal
```